db/migrate/20220823102446_change_type_free_fall_to_jumps.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeTypeFreeFallToJumps < ActiveRecord::Migration[7.0]
+  def up
+    change_table :jumps do |t|
+      t.change :total_freefall, :string
+    end
+
+    def down
+      change_table :jumps do |t|
+        t.change :total_freefall, :integer
+      end
+    end
+  end
```

Comment:
could be as simple as:
```suggestion
  def change
    change_column :jumps, :total_freefall, :string, :integer
  end
```